drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/BRLColumnSubHeaderBuilderDefault.java

```diff
@@ -144,7 +144,7 @@ private GuidedDecisionTable52 makeTempLHSGuidedDecisionTable(final BRLConditionC
         for (int i = 0; i < childColumns.size(); i++) {
             final BRLVariableColumn brlConditionVariableColumn = childColumns.get(i);
             columnContext.addBoundName(brlConditionVariableColumn.getVarName());
-            if (brlConditionVariableColumn.getVarName().isEmpty()) {
+            if (brlConditionVariableColumn.getVarName() == null || brlConditionVariableColumn.getVarName().isEmpty()) {
```

Comment:
3rd issue the file found, but I did not make an ticket for it. BRL columns with no variables caused an NPE here.